site/app/views/grading/ImagesView.php

```diff
@@ -35,16 +35,18 @@ public function listStudentImages($students, $grader_sections, $instructor_permi
         $expected_images_path = FileUtils::joinPaths($this->core->getConfig()->getCoursePath(), "uploads", "student_images");
 
         $dir = new \DirectoryIterator($expected_images_path);
+        //Extensions array can be extended if we want to support more types
+        $valid_image_extensions = array("png", "jpg", "jpeg");
         foreach ($dir as $fileinfo) {
-            if (!$fileinfo->isDot() && $fileinfo->getExtension() === "png") {
+            if (!$fileinfo->isDot() && in_array($fileinfo->getExtension(), $valid_image_extensions)) {
 
                 $expected_image = $fileinfo->getPathname();
                 $content_type = FileUtils::getContentType($expected_image);
                 if (substr($content_type, 0, 5) === "image") {
                     // Read image path, convert to base64 encoding
                     $expected_img_data = base64_encode(file_get_contents($expected_image));
 
-                    $img_name = $fileinfo->getBasename('.png');
+                    $img_name = pathinfo($fileinfo->getBasename(), PATHINFO_FILENAME);
```

Comment:
might be easier to do `$fileinfo->getBasename('.' . $fileinfo->getExtension())`? I dunno what best practice here for PHP is.

Reply:
From my cursory search it seemed like pathinfo() is the suggestion people tend to make, but internet snippets of code doesn't necessarily mean it's best practice. I can change this if you'd like.